src/app/components/header/header.component.ts

```diff
@@ -7,4 +8,24 @@ import { Component } from '@angular/core';
 })
 export class HeaderComponent {
 
+  constructor(private router: Router){}
+
+
+  checkRoute(){
```

Comment:
U don't really need this function, simply access router.url from the HTML and compare it directly against a const that u need

const GAME_VIEW = '/game';
...